pyfr/solvers/navstokes/kernels/bcs/slp-adia-wall.mako

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+<%namespace module='pyfr.backends.base.makoutil' name='pyfr'/>
+
+<%include file='pyfr.solvers.euler.kernels.rsolvers.${rsolver}'/>
+<%include file='pyfr.solvers.navstokes.kernels.bcs.common'/>
+<%include file='pyfr.solvers.navstokes.kernels.flux'/>
+
+<%pyfr:macro name='bc_rsolve_state' params='ul, nl, ur'>
```

Comment:
End up with code duplication and expanding (effectively) the same thing twice in the generated source.  Probably best to remove.